### ISSUE TITLE
Ping method on Bee class

### ISSUE THREAD

**Issue by vojtechsimetka** [ethersphere/bee-js#171]
**Rationale:**
As a user of `Bee` class I would love to check if my connection to the bee node works. To validate that I did not make any configuration error (like wrong URL) or there isn't any network problem (e.g. firewall) before I try to do any real operation (which I would have to handle). This service should not incur any cost as it does not require any interaction with the Swarm network. It maybe should even be socket connection.

**Comment by AuHau** [comment]
Agree! In a perfect world, I would even say that this check should be upon instantiation of `Bee` class, but since it is a class it won't be possible to do that in constructor as this is `async` operation.

What do you mean by "It maybe should even be socket connection."?

**Comment by vojtechsimetka** [comment]
Sorry I meant to write "websocket connection" so that it is a session and can be monitored for any errors or disconnects. But the first most naive implementation is just some REST endpoint.

**Comment by AuHau** [comment]
Yeah, I thought so, but it is just that I am not aware that Bee would expose any websocket... So I guess that is also a prerequisite for that?

**Comment by vojtechsimetka** [comment]
Yes it does (though not for this purpose). The PSS subscribe is a websocket and theoretically we could actually use it for this purpose.

**Comment by agazso** [comment]
> Yes it does (though not for this purpose). The PSS subscribe is a websocket and theoretically we could actually use it for this purpose.

The websocket (PSS feature) is not enabled when running in gateway mode, so it may not be good as a general purpose ping endpoint.

There is also a debug port which we may want to know if it's available or not and it actually has endpoints that are suitable for the purpose: https://docs.ethswarm.org/debug-api/#tag/Status

Keep in mind that all of this is subject to an overhaul: https://github.com/ethersphere/bee/issues/1289

**Comment by agazso** [comment]
For the time being we can use the `/` endpoint, it returns `Ethereum Swarm Bee` as text:

```
% curl -v http://localhost:1633/ 
*   Trying ::1:1633...
* Connected to localhost (::1) port 1633 (#0)
> GET / HTTP/1.1
> Host: localhost:1633
> User-Agent: curl/7.74.0
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Date: Fri, 05 Mar 2021 10:49:53 GMT
< Content-Length: 19
< Content-Type: text/plain; charset=utf-8
< 
Ethereum Swarm Bee
```

**Comment by AuHau** [comment]
@vojtechsimetka I believe this is closed with #207 right? I am closing it, if you think it is not, reopen it, please.
